stdlib/2and3/ctypes/__init__.pyi

```diff
@@ -245,16 +245,16 @@ class Structure(_StructUnionBase): ...
 class BigEndianStructure(Structure): ...
 class LittleEndianStructure(Structure): ...
 
-class Array(Generic[_T], _CData):
+class Array(Generic[_CT], _CData):
     _length_: ClassVar[int] = ...
-    _type_: ClassVar[Type[_T]] = ...
-    raw: bytes = ...  # TODO only available with _T == c_char
-    value: bytes = ...  # TODO only available with _T == c_char
+    _type_: ClassVar[Type[_CT]] = ...
+    raw: Any = ...  # Note: bytes if _CT == c_char, unavailable otherwise
```

Comment:
I understand that `value` needs to be changed, but `raw` is bytes is always `bytes` if it is available, isn't it? In that case the annotation should remain as it is, but the comment fix looks good.

Reply:
You're right, it's either `bytes` or unavailable. I wanted to indicate the "possibly unavailable" part by using `Any` instead of `bytes`, but on second thought it's probably not all that useful (and in the end it would catch fewer type errors).